assets/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "repository": {},
+  "license": "MIT",
+  "scripts": {
+    "deploy": "brunch build --production",
+    "watch": "brunch watch --stdin"
+  },
+  "dependencies": {
+    "phoenix": "file:../deps/phoenix",
```

Comment:
Yea we'll need that for building it on heroku, and also to make it explicit that we're using it. also if we want CI tests we'd need it as a dependency